### PR TITLE
Add link to audio for the flow presentation

### PR DESCRIPTION
**Pull request by alexeyzab**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ The Seller Labs Team
 
 - [ ] Take a picture of yourself and add it to the [company directory](https://sites.google.com/a/roundsphere.com/sellerlabsnet/directory). 
 
-- [ ] Check out this [presentation](https://docs.google.com/a/roundsphere.com/presentation/d/1MEPxa-Z1XGlr2NB12mcWg5K3muQ9l5dNSc7XO-u0IG8/edit?usp=sharing) to walk through the flow of our products, their individual use cases and possibilities, and how they align with our customer's needs. (Accompanying audio recording to come soon!)
+- [ ] Check out this [presentation](https://docs.google.com/a/roundsphere.com/presentation/d/1MEPxa-Z1XGlr2NB12mcWg5K3muQ9l5dNSc7XO-u0IG8/edit?usp=sharing) to walk through the flow of our products, their individual use cases and possibilities, and how they align with our customer's needs. Accompanying audio recording is also [available](https://www.youtube.com/watch?v=0-eSJVQrVS0).
```
Comment (marilyn) [inline README.md]: I might focus this point more on the video now!  We could remove the link to the slides, possibly, or just make it more of a sidenote.

Comment (alexeyzab) [reply README.md]: Makes sense, I'll edit that.

**Comment by alexeyzab** [comment]
@marilyn Swapped the old link with the youtube one and mentioned the slides afterward.

**Comment by marilyn** [comment]
Perfecto, thanks!

**Pull request merged**